app/globals.css

```diff
@@ -0,0 +1,46 @@
+.marker {
```

Comment:
I'm unaware if there is a way to style markers without global CSS at this time, but if we find one, I would recommend migrating to it.

Reply:
that being said, only the address lookup marker and other non-WebGL/non-Layer elements will remain here after moving to Mapbox Layers, so not a priority

Reply:
Moved to #102